webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -273,28 +299,35 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
-.smallTag {
-  width: 100%;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-04T19:05:08Z" title="Wednesday, December 4th 2019, 8:05:08 pm +01:00">Dec 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -172,6 +220,14 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      elem: null,
+      checkedBlur: false,
```


```suggestion
      blurImage: false,
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-06T11:59:03Z" title="Friday, December 6th 2019, 12:59:03 pm +01:00">Dec 6, 2019</time>_

---


then we can keep this `blurImage` and it's not as confusing
`blur` is a verb, so the user is doing and action, so the naming fits better